vickev/howdypix
# Pix 49 send email for a magic link

## Description

First iteration of the authentication 🎊🎊. Adds the following features:

![image](https://user-images.githubusercontent.com/219680/68304604-1793f400-0074-11ea-8eba-b4774e4e40e9.png)
 

- Authentication form that inputs the email
- GraphQL resolver that checks if the email is authorized
- Send the email with the magic link
- New service introduced: maihog to test the sent emails

TODO

- [x] Documentation about the new service
- [x] Jest tests
- [x] debug() 
- [x] Refactor a bit the names that are currently not ideal





## How to test?

- Check-out this branch.
- Run `yarn dev` and go to `http://localhost:3000`
- Enter the following email: `albert@vickev.com`
- Go to `http://localhost:1080/#` to check the emails => should receive the email


## Checklist
